rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/factory/AbstractExamplesFactory.java

```diff
@@ -135,43 +135,43 @@ public Operation generateExample(final Class<? extends Operation> opClass) throw
-            value = (char) uniqueId;
-        } else if (String.class.equals(clazz) || Object.class.equals(clazz)) {
-            value = String.valueOf(uniqueId);
-        } else if (Integer.class.equals(clazz) || int.class.equals(clazz)) {
-            value = uniqueId;
-        } else if (Double.class.equals(clazz) || double.class.equals(clazz)) {
-            value = (double) uniqueId + 0.1;
-        } else if (Long.class.equals(clazz) || long.class.equals(clazz)) {
-            value = (long) uniqueId;
-        } else if (Float.class.equals(clazz) || float.class.equals(clazz)) {
-            value = (float) uniqueId;
-        } else if (Date.class.equals(clazz)) {
-            value = new Date(System.currentTimeMillis() - 10000 + uniqueId);
-        } else if (boolean.class.equals(clazz)) {
-            value = uniqueId % 2 == 0;
-        } else if (BinaryOperator.class.equals(clazz)) {
-            value = new First();
-        } else {
-            try {
-                if (clazz.isEnum()) {
-                    final List l = Arrays.asList(clazz.getEnumConstants());
-                    if (!l.isEmpty()) {
-                        value = Enum.valueOf(clazz, l.get(0).toString());
-                    } else {
-                        value = clazz.newInstance();
-                    }
+        try {
+            if (null == clazz) {
+                value = null;
+            } else if (Character.class.equals(clazz) || char.class.equals(clazz)) {
+                value = (char) uniqueId;
+            } else if (String.class.equals(clazz) || Object.class.equals(clazz)) {
+                value = String.valueOf(uniqueId);
+            } else if (Integer.class.equals(clazz) || int.class.equals(clazz)) {
+                value = uniqueId;
+            } else if (Double.class.equals(clazz) || double.class.equals(clazz)) {
+                value = (double) uniqueId;
+            } else if (Long.class.equals(clazz) || long.class.equals(clazz)) {
+                value = (long) uniqueId;
+            } else if (Float.class.equals(clazz) || float.class.equals(clazz)) {
+                value = (float) uniqueId;
+            } else if (Date.class.equals(clazz)) {
+                value = new Date(System.currentTimeMillis() - 10000 + uniqueId);
+            } else if (boolean.class.equals(clazz)) {
+                value = uniqueId % 2 == 0;
+            } else if (BinaryOperator.class.equals(clazz)) {
+                value = new First();
+            } else if (clazz.isEnum()) {
+                final List l = Arrays.asList(clazz.getEnumConstants());
+                if (!l.isEmpty()) {
+                    value = Enum.valueOf(clazz, l.get(0).toString());
                 } else {
                     value = clazz.newInstance();
                 }
-            } catch (final InstantiationException | IllegalAccessException e) {
-                value = null;
+            } else {
+                value = clazz.newInstance();
```

Comment:
This is ok for Java 8, but [for Java 9+ it is deprecated](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Class.html#newInstance()). The recomended solution is compatible with Java 8 so I'd suggest using it in these changes.